openshift/unleash.yaml

```diff
@@ -148,6 +150,29 @@ objects:
         name: ${identifier}
     selector:
       app: ${identifier}
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    annotations:
+      cert-manager.io/issuer-kind: ClusterIssuer
+      cert-manager.io/issuer-name: letsencrypt-prod-http
```

Comment:
why adding this annotations now? I'm just worried that in a moment we have both cert-manager in the cluster AND openshift-acme in the namespace...

Reply:
I took it form the current a-i route.

Reply:
cert-manager is the new kid in town (https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/43592/diffs)

Reply:
I've removed the `openshift-acme` annotation.